src/com/main/db/Database.java

```diff
@@ -143,135 +145,137 @@ public List<Item> fetchItems(int vendorId) {
 				i.setQuantity(quantity);
 				li.add(i);
 			}
-		}catch(SQLException e) {
+		} catch (SQLException e) {
 			e.printStackTrace();
 		}
 		return li;
 	}
 
 	/*
 	 * KEVIN
+	 * 
 	 * @param vendor - Vendor obj to be inserted
 	 */
 	public void addVendor(Vendor vendor) {
 		dbConnect();
-		
-		String sql="insert into vendor(businessId,name,username,password) values (?,?,?,?);";
-		 
+
+		String sql = "insert into vendor(businessId,name,username,password) values (?,?,?,?);";
+
 		try {
 			PreparedStatement pstmt = con.prepareStatement(sql);
 			pstmt.setInt(1, vendor.getBusinessId());
 			pstmt.setString(2, vendor.getName());
 			pstmt.setString(3, vendor.getUsername());
 			pstmt.setString(4, vendor.getPassword());
-			
+
 			pstmt.executeUpdate();
-		
+
 		} catch (SQLException e) {
 			e.printStackTrace();
 		}
-		
+
 		dbClose();
 	}
-	
+
 	/** ORDER */
-	
+
 	/*
-	 * KEVIN
-	 * TO BE CALLED WHEN CUSTOMER ADDS THEIR FIRST ITEM FROM A VENDOR
+	 * KEVIN TO BE CALLED WHEN CUSTOMER ADDS THEIR FIRST ITEM FROM A VENDOR
```

Comment:
Why're you moving all the method descriptions into the same line as the author name?